src/yape06/yape06.h

```diff
@@ -33,6 +33,9 @@ class Yape06 {
     assert(border > 0 && "border value must be > o!");
     auto points = pts->kpoints;
     auto out = outPts->kpoints;
+    Yape06Points ypts;
```

Comment:
we need to solve this hack. This of course works only if we have that size of Keypoints, we need instead to pass a reference of the input KeyPoints to init the internal KeyPoints in the struct.